api/v1beta1/bgppeer_webhook.go

```diff
@@ -137,6 +137,13 @@ func (bgpPeer *BGPPeer) validateBGPPeersRouterID(existingBGPPeersList *BGPPeerLi
 		return field.Invalid(field.NewPath("spec").Child("RouterID"), routerID,
 			fmt.Sprintf("Invalid RouterID %s", routerID))
 	}
+	for _, BGPPeer := range existingBGPPeersList.Items {
```

Comment:
If this only relates to FRR I think we should use some conditional here to allow it for native.

Reply:
+1